assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js

```diff
@@ -55,47 +57,67 @@ export const TEXT = {
 function GoogleAnalyticsSettings({ googleAnalyticsId, handleUpdate }) {
   const [analyticsId, setAnalyticsId] = useState(googleAnalyticsId);
   const [inputError, setInputError] = useState('');
+  const canSave = analyticsId !== googleAnalyticsId && !inputError;
+  const disableSaveButton = !canSave;
 
   useEffect(() => {
     setAnalyticsId(googleAnalyticsId);
   }, [googleAnalyticsId]);
 
-  const handleCancelUpdateId = useCallback(() => {
-    setAnalyticsId(googleAnalyticsId);
-  }, [googleAnalyticsId, setAnalyticsId]);
+  const handleUpdateId = useCallback((event) => {
+    const { value } = event.target;
+    setAnalyticsId(value);
+
+    if (value.length === 0 || validateGoogleAnalyticsIdFormat(value)) {
+      setInputError('');
+
+      return;
+    }
 
-  const handleUpdateId = useCallback(
-    (value) => {
-      if (value.length === 0 || validateGoogleAnalyticsIdFormat(value)) {
-        setInputError('');
-        return handleUpdate(value);
+    setInputError(TEXT.INPUT_ERROR);
+  }, []);
+
+  const handleOnSave = useCallback(() => {
+    if (canSave) {
+      handleUpdate(analyticsId);
+    }
+  }, [canSave, analyticsId, handleUpdate]);
+
+  const handleOnKeyDown = useCallback(
+    (e) => {
+      if (e.key === 'Enter') {
+        e.preventDefault();
+        handleOnSave();
       }
-      return setInputError(TEXT.INPUT_ERROR);
     },
-    [handleUpdate, setInputError]
+    [handleOnSave]
   );
+
   return (
     <SettingForm onSubmit={(e) => e.preventDefault()}>
       <SettingHeading htmlFor="gaTrackingID">
         {TEXT.SECTION_HEADING}
       </SettingHeading>
       <FormContainer>
-        <InlineInputForm
+        <TextInput
           label={TEXT.ARIA_LABEL}
           id="gaTrackingId"
           value={analyticsId}
-          onEditCancel={handleCancelUpdateId}
-          onEditComplete={handleUpdateId}
+          onChange={handleUpdateId}
+          onKeyDown={handleOnKeyDown}
           placeholder={TEXT.PLACEHOLDER}
           error={inputError}
-          noAutoFocus={true}
         />
+        {inputError && <ErrorText>{inputError}</ErrorText>}
         <TextInputHelperText>
           {TEXT.CONTEXT}
           <InlineLink href={TEXT.CONTEXT_ARTICLE_LINK}>
             {TEXT.CONTEXT_ARTICLE}
           </InlineLink>
         </TextInputHelperText>
+        <SaveButton isDisabled={disableSaveButton} onClick={handleOnSave}>
+          {__('Save', 'web-stories')}
+        </SaveButton>
```

Comment:
Sam left updated designs here: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories---Stable-Release?node-id=2%3A89741 on 9/1/2020 - https://github.com/google/web-stories-wp/issues/3843 

Since this does solve the issue of not knowing how to save the analytics ID and since the new button styles haven't been replaced in the dashboard I'm in favor of splitting the difference with what's in the designs and what's here and just moving the save button up above the helper text so when you tab out of the input it's the first thing you focus on. we can adjust the button styling and shared line as a follow next sprint.